packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -202,12 +202,12 @@ module.exports = class SingleSpaReactGenerator extends Generator {
         coloredFinalInstructions(
           `1. Run '${this.options.packageManager} start${
             this.options.packageManager === "npm" ? " --" : ""
-          } --https --port 8500'`
+          } --port 8500'`
         )
       );
       console.log(
         coloredFinalInstructions(
-          `2. Go to https://single-spa-playground.org/playground/instant-test?name=@${this.orgName}/${this.projectName}&url=8500 to see it working!`
+          `2. Go to http://single-spa-playground.org/playground/instant-test?name=@${this.options.orgName}/${this.options.projectName}&url=8500 to see it working!`
```

Comment:
We changed the orgName and projectName to be options in a recent release, but forgot to update this.